roles/keystone/tasks/main.yml

```diff
@@ -89,11 +107,19 @@
   notify: restart keystone services
   # we want this to always be changed so that it can notify the service restart
   tags: db-migrate
-
+  
+- name: enable keystone apache site
+  apache2_site: name=keystone state=enabled
+  notify:
+    - reload apache
+    
 - meta: flush_handlers
 
 - name: start keystone
   service: name=keystone state=started
+  
+- name: start keystone (apache)
+  service: name=apache2 state=started
```

Comment:
I just need to wrap my head around it.  Front end means that it takes the incoming http connection, and routes it to the running uwsgi process on the socket? So to stop the actual horizon service, you'd stop the uwsgi service, and then apache can't complete the routing?